go.mod

```diff
@@ -26,10 +26,10 @@ require (
 	github.com/oklog/run v1.1.0
 	github.com/open-policy-agent/opa v0.39.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus-community/prom-label-proxy v0.3.1-0.20210623095334-9d425172d7bb
+	github.com/prometheus-community/prom-label-proxy v0.6.0
```

Comment:
Anything else we need to be aware by jumping so many versions? Anything particular that your eye catched scanning the changelog?

Reply:
These ones kinda popped out from the [changelog](https://github.com/prometheus-community/prom-label-proxy/blob/main/CHANGELOG.md):

```
v0.4.0
- [ENHANCEMENT] Support HTTP POST for /api/v1/labels endpoint.
- [ENHANCEMENT] Add label enforce support for the new query_exemplars API.

v0.5.0
- [ENHANCEMENT] Add /healthz endpoint for (Kubernetes) probes.

v0.6.0
- [FEATURE] Add the --header-name flag to pass the label value via HTTP header.
- [FEATURE] Add the --internal-listen-address flag to expose Prometheus metrics. 
```

Reply:
:+1:

Reply:
Feel free to tag me in the future as I'm maintaining prom-label-proxy. Just to confirm: the bump should be safe.